README-sycl.md

```diff
@@ -15,16 +15,82 @@ code for heterogeneous and offload processors to be written with modern
 ISO C++, and provides API and abstractions to find devices and manage 
 resources for GPUs. 
 
-## What works
+## Support for NVIDIA GPUs using CUDA
 
-Currently, only one example works on NVIDIA SM 80.
+The support for NVIDIA GPUs using CUTLASS is unmodified, you can still use this repository as a drop-in replacement of the upstream NVIDIA repository.
+The SYCL support does not conflict with the original NVIDIA CUDA path.
+Only some portions of the common headers and the build system are slightly modified
+to enable the SYCL compilation mode.
 
-## Requirements
+We aim to integrate any changes from the upstream NVIDIA repository as soon
+as we can.
+
+## Support for Intel GPUs
+
+The SYCL backend supports running CUTLASS on Intel GPUs.
+Currently, Intel Data Center Max 1550 and 1100 (a.k.a PVC) are supported.
+Intel Arc B570 is known to work but not optimized yet.
+
+The sycl folder within the examples directory shows a number of GEMM algorithms and examples of CUTLASS running on PVC, including flash attention V2.
```

Comment:
```suggestion
The `examples/sycl` directory shows a number of GEMM algorithms and examples of CUTLASS running on PVC, including flash attention V2.
```

Reply:
Addressed in 92faba5acf09505e2f7b1192b0d9d98e1baa9468